nvsvocprez/app.py

```diff
@@ -1517,11 +1519,17 @@ def _render_nvs_or_profile_html(self):
                 FILTER ( ?p != skos:related )
                 FILTER ( ?p != owl:sameAs )
             """
+        excluded_profiles = []
         for alt in self.alt_profiles.values():
+            ontology_prefixes = alt["ontology_prefix"].split(",")
```

Comment:
For this can we not just get the current profile using self.profiles[self.profile]? I've altered the profile object to hold a dict of ontologies associated with that view.

So then you can generate prefix's based on ontologies listed in current_profile.ontologies, and then generate filters by looping through self.ontologies (all ontologies) and checking which ones aren't present in current_profile.ontologies?

Hope that made sense..